gustaf/utils/init_helper.py

```diff
@@ -0,0 +1,100 @@
+from typing import Any
+
+
+class SplineCanNotBeLoadedHelper():
+    """
+    Class used to have better import error handling in the case the 'splinepy'
+    package is not installed. This is necessary due to that `splinepy` is not
+    a requirement of `gustaf`, but some parts of `gustaf` need it to function.
+
+    This class makes it so that `gustaf` is able to provide all non-spline
+    based functionality with out `splinepy`, but gives also comprehensive
+    errors in case these functionalities are used without the required package.
+    """
+
+    def __call__(self, *args: Any, **kwds: Any) -> Any:
+        """
+        Is called when the object is called by object(). Will notify the user,
+        that the functionality is not accessible and how to proceed to access
+        the functionality.
+        """
+        raise ImportError(
+            "This gustaf functionality can not be provided due to not being "
+            "able to load the `splinepy` package. Please see the install "
+            "instructions on how to install it."
+        )
```

Comment:
```suggestion
        raise ImportError(self.__message)
```

Reply:
Just out of curiosity, whats the motivation to use `pass` here

Reply:
artifact from development is gone in the reworked version